### PR TITLE
Add translation strings and `hu` translation

### PR DESCRIPTION
**Pull request by An0n3m0us**
Changes strings to translation strings so one can create translations for the mod.
I also added a template translation file and Hungarian translation.

### CODE REVIEW

File: cobble.lua [hunk 1]
```diff
@@ -1,3 +1,6 @@
+-- Load support for MT game translation.
+local S = minetest.get_translator("default")
```
Comment (louisroyer) [inline cobble.lua]:
```suggestion
local S = minetest.get_translator("more_chests")
```

File: locale/more_chests.hu.tr [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# textdomain: default
```
Comment (louisroyer) [inline locale/more_chests.hu.tr]:
```suggestion
# textdomain: more_chests
```

File: secret.lua [hunk 3]
```diff
@@ -1,3 +1,6 @@
+-- Load support for MT game translation.
+local S = minetest.get_translator("default")
```
Comment (louisroyer) [inline secret.lua]:
```suggestion
local S = minetest.get_translator("more_chests")
```

File: shared.lua [hunk 4]
```diff
@@ -1,3 +1,6 @@
+-- Load support for MT game translation.
+local S = minetest.get_translator("default")
```
Comment (louisroyer) [inline shared.lua]:
```suggestion
local S = minetest.get_translator("more_chests")
```

File: wifi.lua [hunk 5]
```diff
@@ -1,5 +1,8 @@
+-- Load support for MT game translation.
+local S = minetest.get_translator("default")
```
Comment (louisroyer) [inline wifi.lua]:
```suggestion
local S = minetest.get_translator("more_chests")
```

File: locale/template.txt [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# textdomain: default
```
Comment (louisroyer) [inline locale/template.txt]:
```suggestion
# textdomain: more_chests
```

File: cobble.lua (hunk 1 above)
Comment (An0n3m0us) [reply cobble.lua]: Fixed.

File: locale/more_chests.hu.tr (hunk 2 above)
Comment (An0n3m0us) [reply locale/more_chests.hu.tr]: Fixed.

File: secret.lua (hunk 3 above)
Comment (An0n3m0us) [reply secret.lua]: Fixed.

File: shared.lua (hunk 4 above)
Comment (An0n3m0us) [reply shared.lua]: Fixed.

File: wifi.lua (hunk 5 above)
Comment (An0n3m0us) [reply wifi.lua]: Fixed.

File: locale/template.txt (hunk 6 above)
Comment (An0n3m0us) [reply locale/template.txt]: Fixed.

**Comment by An0n3m0us** [comment]
@louisroyer OK done; I tried to match your PR as much as possible so we don't get merge conflicts.
And I don't think minetest.log outputs are translatable; at least they don't seem to be.

**Comment by louisroyer** [comment]
Uh, you are right, I will remove log output from translation files. Sorry for the extra translation work you done because of this.

**Comment by An0n3m0us** [comment]
@louisroyer It's alright; it was easy to translate.

**Comment by An0n3m0us** [comment]
Fix merge conflicts.

**Pull request merged**